README.md

```diff
@@ -0,0 +1,30 @@
+# Installation
+This installation assumes you have a running instance of sensu-base and for ssl that you have your ssl-certs generated by the sensu-tool.
+```
+juju deploy sensu-client
+```
+The sensu-client requires 2 config options, password, which is the vhost password for the rabbitmq-server, exposed in juju status. The other one is rabbitmq, which is the privateip:port of the rabbitmq-instance. When using ssl, 2 extra config options must be set. Assuming you are in the sensu_ssl_tool directory:
+```
+juju config sensu-client rabbitmq="private_ip:port"
+juju config sensu-client password="[rabbitmq password]"
+juju config sensu-client ssl_key="`cat client/key.pem`" ssl_cert="`cat client/cert.pem`"
```

Comment:
As mentioned in the beginning from the installation, these keys are generated by the sensu_ssl_tool. Since these keys must be the same for the rabbitmq, sensu-base and sensu-client, generating them inside a charm is not possible. EasyRSA integration is also not an option for now, since rabbitmq does not support it.